src/odoc/resolver.ml

```diff
@@ -580,13 +579,21 @@ let create ~important_digests ~directories ~open_modules ~roots =
               { Named_roots.name; dir; omit })
             roots
         in
+        let directories =
+          match current_package with
+          | None -> directories
+          | Some pkg -> (
+              try List.assoc pkg page_roots :: directories
+              with _ -> directories)
```

Comment:
This is a change for drivers... If I understand correctly:
- Any `-L _:<path>` also adds `<path>` to the search path
- A `-P pkg:<path>` adds `<path>` to the search path _when_ `pkg` is the current package

I think this should be added to the odoc manpage and the driver doc page, now that it is merged!

Reply:
Yes, that's the behaviour I intended. What I wasn't sure about was whether this is the right mechanism to do this?

Reply:
Seems reasonable... Not adding anything to the search path would be another reasonable approach (that would give more work, but more freedom, to the driver).